### PR TITLE
 "Revert "Updated Decimal to be a java.lang.Number and to have a bit more utili…""

### PR DESCRIPTION
**Pull request by team172011**
Reverts ta4j/ta4j#104
The test that proves multiplication symmetrically has failed.

### CODE REVIEW

File: ta4j-core/src/main/java/org/ta4j/core/Decimal.java [hunk 1]
```diff
@@ -72,24 +74,28 @@ private Decimal(String val) {
...
+        delegate = new BigDecimal(val, MATH_CONTEXT);
+    }
+
+    private Decimal(float val) {
+        delegate = new BigDecimal(val, MATH_CONTEXT);
+    }
+
+    private Decimal(double val) {
+        delegate = new BigDecimal(val, MATH_CONTEXT);
     }
```
Comment (team172011) [inline ta4j-core/src/main/java/org/ta4j/core/Decimal.java:95]: Please use static function `valueOf(val)` to generate new Decimal instances. See  #84 and #86

Comment (briandilley) [reply ta4j-core/src/main/java/org/ta4j/core/Decimal.java:95]: This is a constructor

Comment (briandilley) [reply ta4j-core/src/main/java/org/ta4j/core/Decimal.java:95]: Oh, nevermind - i see what you mean.

**Comment by briandilley** [comment]
I can't get the `DecimalTransformIndicatorTest` to fail locally.  However, `PivotPointIndicatorTest` is failing for something that i feel is unrelated.

**Comment by briandilley** [comment]
Fuck this PR, use this one instead: https://github.com/ta4j/ta4j/pull/106

**Comment by team172011** [comment]
moved to #106

**Pull request closed without merging**